.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: test
+on:
+  pull_request:
+    branches: [ main ]
+jobs:
+  tests:
+    name: Acceptance Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
+    strategy:
+      fail-fast: false
+      matrix:
+        parallel: [3]
+        index: [0, 1, 2]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+          cache: true
+
+      - name: Sync dependencies
+        run: |
+          go get -v -t -d ./...
+          go mod tidy
```

Comment:
If go mod tidy fails does it fail the build?

Reply:
Yes. Any nonzero exit value on any step will cause the workflow to fail. In this case, the other two runs the matrix will continue because I specified "fail-fast: false" so that they can be retried individually.

Reply:
I moved the fmtcheck and vet steps to the lint workflow.